e2e/cypress/integration/enterprise/guest_accounts/guest_experience_ui_spec.js

```diff
@@ -217,4 +217,29 @@ describe('Guest Account - Guest User Experience', () => {
             });
         });
     });
+
+    it('MM-T1412 Revoke Guest User Sessions when Guest feature is disabled', () => {
+        // # Demote Guest user if applicable
+        demoteGuestUser(guestUser);
+
+        // # Disable Guest Access
+        cy.apiUpdateConfig({
+            GuestAccountsSettings: {
+                Enable: false,
+            },
+        });
+
+        // # Wait for page to load and then logout
+        cy.get('#post_textbox').should('be.visible').wait(TIMEOUTS.TWO_SEC);
+        cy.apiLogout();
+        cy.visit('/');
+
+        // # Login with guest user credentials and check the error message
+        cy.get('#loginId').type(guestUser.username);
+        cy.get('#loginPassword').type('passwd');
+        cy.findByText('Sign in').click();
+
+        // # Verify if guest account is deactivated
```

Comment:
Really minor: // *

Reply:
@srkgupta Feel free to merge once this comment is resolved.